brainiak/searchlight/searchlight.py

```diff
@@ -24,6 +25,73 @@
 ]
 
 
+class Shape:
+    """Shape
+
+    Searchlight shape which is contained in a (2*rad+1)^3 cube
+    """
+    def __init__(self, rad):
+        """Constructor
+
+        Parameters
+        ----------
+
+        rad: radius, in voxels, of the sphere inscribed in the
+             searchlight cube, not counting the center voxel
+
+        """
+        self.rad = rad
+
+
+class Cube(Shape):
+    """Cube
+
+    Searchlight shape which is a (2*rad+1)^3 cube
+    """
+    def __init__(self, rad):
+        """Constructor
+
+        Parameters
+        ----------
+
+        rad: radius, in voxels, of the sphere inscribed in the
+             searchlight cube, not counting the center voxel
+
+        """
+        super(Cube, self).__init__(rad)
```

Comment:
You should call `super()` with no arguments (also in `Diamond`):
https://docs.python.org/3/library/functions.html#super